### ISSUE TITLE
[TypeSpec Validation] Create support team and recommend it when check fails

### ISSUE THREAD

**Issue by mikeharder** [Azure/azure-rest-api-specs#25229]
* [ ] Create a GitHub team like `@Azure/tsv-support` spec authors can mention for support when check fails
* [ ] Update check to recommend mentioning team when check fails (may depend on #24999)

**Comment by mikeharder** [comment]
Closing in favor of https://github.com/Azure/azure-rest-api-specs-pr/issues/14250
